appgate/openapi/openapi.py

```diff
@@ -209,6 +209,19 @@ def add_items_key_to_tags(obj: dict) -> dict:
 
     schema = add_items_key_to_tags(schema)
 
+    # replace underscore with period
+    def replace_underscore(obj: dict) -> dict:
```

Comment:
Just a nitpick but seems like the comment could a docstring instead, also mention its recursive.
```suggestion
    def replace_underscore(obj: dict) -> dict:
        """
        Replace underscore with period recursively.
        """
```